### PR TITLE
Feature to brings next word

### PR DESCRIPTION
**Pull request by lucascomputacao**
I'm trying to learn more about frontend and trying to add some features on this project during the process. In this PR I added a simple button to "brings" another word (you can see on the screenshot).
Only this! =)
screen shot 2018-08-21 at 19 39 42

### CODE REVIEW

**Comment by henriqueboaventura** [comment]
Can I suggest something? Don't you think that the button would be better if the background was the same colour as the border? Maybe it would highlight it more :)

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -36,11 +36,23 @@ video {
...
+    font-size: 16px;
+    margin-bottom: 50px;
+}
+
 footer {
     color: #2C3E50;
     font-size: 0.8em;
     width: 100%;
-    position: absolute;
+    /* position: absolute;` */
```
Comment (henriquelbsouza) [inline public/stylesheets/style.css]: Can we remove this commented line?

File: routes/index.js [hunk 2]
```diff
@@ -5,21 +5,20 @@ var request = require('request');
 /* GET home page. */
 router.get('/', function(req, res, next) {
   signalId = Math.floor(Math.random() * (5823 - 1) + 1);
+  console.log("request: 'http://www.acessibilidadebrasil.org.br/libras_3/ajax/getWordById/" + signalId + "'");
```
Comment (henriquelbsouza) [inline routes/index.js]: Is it still needed?

**Review by henriquelbsouza — COMMENTED** [review]
Apart from dead code/debug logs, it is fine for me. Just take a look on @kidh0's comment about layout stuff :)

File: routes/index.js (hunk 2 above)
Comment (lucascomputacao) [reply routes/index.js]: No, I saw after and already remove it! =)

File: public/stylesheets/style.css (hunk 1 above)
Comment (lucascomputacao) [reply public/stylesheets/style.css]: Yeah, Sure!

**Comment by lucascomputacao** [comment]
I maked the changes cited in PR. So when you get some free time, take a look it, plz!

**Comment by henriqueboaventura** [comment]
@lucascomputacao there still a console.log :(

**Comment by lucascomputacao** [comment]
The "console.log" on line 10 of routes/index.js already was there, but I removed it =). I think this the change asked, am I correct?

**Comment by henriqueboaventura** [comment]
Done!

**Pull request merged**